pkg/graph/resolvers/plan_discussion.go

```diff
@@ -454,13 +454,24 @@ func CreateDiscussionReply(
 		if err != nil {
 			return reply, err
 		}
+
 		// Create Activity and notifications in the DB
-		_, notificationErr := notifications.ActivityTaggedInDiscussionReplyCreate(ctx, tx, principal.Account().ID, discussion.ModelPlanID, discussion.ID, reply.ID, reply.Content, loaders.UserNotificationPreferencesGetByUserID)
+		_, notificationErr := notifications.ActivityTaggedInDiscussionReplyCreate(ctx, tx, discussion.CreatedBy, discussion.ModelPlanID, discussion.ID, reply.ID, reply.Content, loaders.UserNotificationPreferencesGetByUserID)
```

Comment:
This method call was updated incorrectly, `principal.Account().ID` is the actor, you switched it to `discussion.CreatedBy`. Can you please revert the change you made to this line?